src/rpc/net.cpp

```diff
@@ -440,6 +439,7 @@ UniValue getnetworkinfo(const JSONRPCRequest& request)
             "  \"localrelay\": true|false,              (bool) true if transaction relay is requested from peers\n"
             "  \"timeoffset\": xxxxx,                   (numeric) the time offset\n"
             "  \"connections\": xxxxx,                  (numeric) the number of connections\n"
+            "  \"maxconnections\": xxxxx,               (numeric) the number of connections\n"
```

Comment:
This description is the same as the one for 'connections' on the previous line. Perhaps "(numeric) the number of max connections\n" instead to differentiate it.
```suggestion
            "  \"maxconnections\": xxxxx,               (numeric) the number of max connections\n"
```